### PR TITLE
restyle homepage signpost row and add containers

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Added containers to the homepage signpost row and restyled the row.
Also tidied up the Sass to use Sass features and BEM'd it.

## QA

View the homepage in all viewports
[Demo](http://www.ubuntu.com-homepage-update.demo.haus/)
[Design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/homepage/03_2017/Exportables/old?preview=Homepage_03_2017.jpg)

### CODE REVIEW

**Comment by deadlight** [comment]
[Add containers icon to homepage](https://trello.com/c/ZKihVYnm/2738-add-containers-icon-to-homepage)

**Comment by deadlight** [comment]
@joanasa89 demo updated

File: static/css/section/_homepage.scss [hunk 1]
```diff
@@ -141,75 +141,74 @@ body.homepage {
   }
 }
 
-.row--intro {
-  background: $ubuntu-orange;
-  color: #fff;
-}
+// row--intro / ubuntu-intro
+$full-size: 1020px;
```
Comment (WillMoggridge) [inline static/css/section/_homepage.scss]: Would this be better in the settings file so we can keep track of them?

Comment (deadlight) [inline static/css/section/_homepage.scss]: @WillMoggridge it's a very specific measurement to this component. I think that it's cleaner here.

**Pull request merged**